cookbook/Berksfile

```diff
@@ -0,0 +1,8 @@
+source 'https://supermarket.chef.io'
+
+metadata
+
+cookbook 'java'
+cookbook 'et_nginx'
```

Comment:
agreed, the community ones are a bit behind sometimes, just pointing it out because a berks install might fail in this case